README.md

```diff
@@ -716,6 +716,85 @@ The test library includes custom `ViewMatchers` and `ViewActions` for easier tes
 
 ## Releasing
 
-In order to release the library, you need to provide a Github personal access token with `repo` scope via a `GITHUB_TOKEN` environment variable - you can create a new token on [Github](https://github.com/settings/tokens/new).
+### Setup - Github personal access token
 
-To release the library, run `bundle exec fastlane tag_release` and choose to bump either the major, minor or patch when prompted. This will create a new tag, update the changelog and trigger the `publish` workflow on Bitrise.
+In order to release the library, you need to provide a Github personal access token with the following scope:
```

Comment:
I needed to do this step just to build, so I think changing the wording away from 'in order to release' might be clearer as it is needed for the earlier steps too..

Reply:
Good point - I've updated it in eb36e3f8623c7bc13282dd21ba5752465cb3745f